src/picongpu/include/traits/PICToAdios.tpp

```diff
@@ -33,6 +33,20 @@ namespace picongpu
 namespace traits
 {
     template<>
+    struct PICToAdios<bool>
+    {
+        ADIOS_DATATYPES type;
+
+        PICToAdios() :
+        type(adios_unsigned_byte) {}
```

Comment:
Thanks for the question. It's an enum, or in-code an int.
There is no need to modify that, since this is in a non-sensitive part of the code. The only sensitive part on the host side is data movement and data re-processing, besides that we needs such tunings on the device. For all other places, like here, usability and maintainability first since it does not matter for performance.

It would also unnecessarily refactor the already existing interface we have right now which is sufficient.